_docs/example-collections.md

```diff
@@ -562,13 +562,13 @@ JSON_END
 
 CODE_START
 CREATE_START details PayinMethodDetails
-SET LIT phone_number "+256301234567"
-SET LIT send_instructions true
+SET LIT phone_number "+2339999999"
+SET LIT mobile_provider "vodafone"
 CREATE_END
 
 CREATE_START method PayinMethod
 SET LIT type "GHS::Mobile"
-SET LIT provider "interpay"
+SET LIT ux_flow "ussd_popup"
```

Comment:
ux_flow is an enum field, so this should be an enum literal. See other enums in the codebase for example how to use it